pkg/commands/git.go

```diff
@@ -159,9 +159,7 @@ func NewGitCommandAux(
 	stashCommands := git_commands.NewStashCommands(gitCommon, fileLoader, workingTreeCommands)
 	patchBuilder := patch.NewPatchBuilder(cmn.Log,
 		func(from string, to string, reverse bool, filename string, plain bool) (string, error) {
-			// TODO: make patch builder take Gui.IgnoreWhitespaceInDiffView into
-			// account. For now we just pass false.
-			return workingTreeCommands.ShowFileDiff(from, to, reverse, filename, plain, false)
+			return workingTreeCommands.ShowFileDiff(from, to, reverse, filename, plain)
```

Comment:
Previously it was possible to render a diff with plain=true and ignoreWhitespace=true. This is no longer possible, plain=true now implies ignoreWhitespace=false. I _think_ this is fine, but double-checking this would be good.

Reply:
This sounds fine to me. You shouldn't be hiding whitespace when doing stuff with custom patches (which use plain diffs) because it would be impossible to apply the diff (I'm pretty sure)